presto-docs/src/main/sphinx/release/release-0.253.rst

```diff
@@ -0,0 +1,32 @@
+=============
+Release 0.253
+=============
+
+**Highlights**
+==============
+
+**Details**
+===========
+
+General Changes
+_______________
+* Fix amplified `Cumulative User Memory` metrics calculation.
+* Fix the version name displayed in the UI.
+* Add tracking for system memory used by column statistics in TableFinishOperator.
+* Add support for shutting down coordinator via `/v1/info/state` endpoint.
```

Comment:
I think we need double quotes here. Can you try generating the docs offline and see if this highlighted.